iot-hub/Samples/service/ServiceClientSample/ServiceClientSample.csproj

```diff
@@ -1,12 +1,13 @@
-<Project Sdk="Microsoft.NET.Sdk">
+﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
     <OutputType>Exe</OutputType>
     <TargetFramework>netcoreapp2.1</TargetFramework>
+    <LangVersion>8</LangVersion>
   </PropertyGroup>
 
   <ItemGroup>
-  	<PackageReference Include="Microsoft.Azure.Devices" Version="1.*" />

```

Comment:
what version was it picking up if it was "*"? 

& so does this mean we change references in sample project with each release?

Reply:
Whatever the latest was for that person when they built the project.

We don't have to rev our samples' references on each release. Each release we have stands the test of time for a while. We can choose to update it on occasion if we have new API surface that we want to illustrate, though.

Reply:
yes but then we need to remember to do it. One extra step to do for releases. We should at least add it in our release notes.

Reply:
We don't need to update it for each release. There is no such requirement.